_posts/2020-11-17-dropping-puppet-5-and-RHEL-6-support.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Dropping Puppet 5 and RHEL 6 support
+date: 2020-03-29
+github_username: bastelfreak
+twitter_username: BastelsBlog
+---
+
+Hi Vox Pupuli followers!
+Puppet 5 will be end of life in the next days, Puppet 7 is around the corner.
+Also RHEL/CentOS will be end of life in the next few days. In the next few days
```

Comment:
instead of repeating in the next few days, suggest listing the dates that each is EOL and stating that after that date, support will drop as well as functional testing.